### PR TITLE
fix small typos

### PR DESCRIPTION
**Pull request by MiConnell**
## Description
update references of withdrawal (when used as a verb) to withdraw

### CODE REVIEW

File: src/content/staking/withdrawals/index.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ As an alternative to changing the withdrawal address for a particular validator,
 <p>Once a validator has completed the exiting process, and assuming the account has withdrawal credentials, the remaining balance will <em>then</em> be withdrawn during the next validator sweep.</p>
 </ExpandableCard>
 
-<ExpandableCard title="How can I withdrawal a custom amount?">
+<ExpandableCard title="How can I withdraw a custom amount?">
```
Comment (minimalsm) [inline src/content/staking/withdrawals/index.md:146]: @wackerow I think the current open wording implies that custom amount withdrawals are possible. I'd personally phrase this as a closed question. WDUT?

```suggestion
<ExpandableCard title="Can I withdraw a custom amount?">
```

**Review by minimalsm — APPROVED** [review]
Thanks @MiConnell ❤️

Suggested a small improvement to the page, but it doesn't need to block this PR merging.

File: src/content/staking/withdrawals/index.md (hunk 1 above)
Comment (MiConnell) [reply src/content/staking/withdrawals/index.md:146]: I agree this wording is better

**Review by corwintines — APPROVED** [review]

File: src/content/staking/withdrawals/index.md [hunk 2]
```diff
@@ -166,10 +166,9 @@ eventName="read more">
 <p>Once a validator has completed the exiting process, and assuming the account has withdrawal credentials, the remaining balance will <em>then</em> be withdrawn during the next validator sweep.</p>
 </ExpandableCard>
 
-<ExpandableCard
-title="How can I withdrawal a custom amount?"
+<ExpandableCard title="Can I withdraw a custom amount?">
```
Comment (corwintines) [inline src/content/staking/withdrawals/index.md:169]:
```suggestion
<ExpandableCard title="Can I withdraw a custom amount?"
```

**Pull request merged**